ocp4-logging/README.md

```diff
@@ -0,0 +1,37 @@
+# Deploying Openshift v4.X Cluster-Logging Stack
+
+## Overview
+
+This quickstart helps you deploy the cluster-logging stack for Openshift 4.X. This is not deployed by default and is accomplished through the subscribing to 2 operators via OperatorHub and then creating the appropriate custom resource.
+
+This example deploys a 1 Elasticsearch and 1 Kibana EFK stack. This can be modified by updating the `ELASTICSEARCH_NODE_COUNT` and `KIBANA_NODE_COUNT` parameters in your `openshift-applier` inventory which can be found [here](.applier/group_vars/seed-hosts.yml).
+
+🚨If deploying a multi-node Elasticsearch cluster, make sure to also update the `ELASTICSEARCH_REDUNDANCY_POLICY` parameter to SingleRedundancy🚨
+
+If you're interested in the other parameters for this logging stack, you can see them [here](.openshift/templates/README.md)
+
+## OCP Objects
+
+This quickstart will create the following:
+
+- A namespace called `openshift-logging` with the appropriate labels
+- One OperatorGroup: ``
+- Two `CatalogSourceConfig` objects: `` and ``
```

Comment:
maybe link to docs about what these objects are

Reply:
Updated with where these objects are created and links to the specific resources. Can't find a good one to link to Subscriptions though.